socorro/unittest/external/boto/test_crash_data.py

```diff
@@ -35,14 +34,13 @@ def test_get_processed(self, boto_helper):
 
     @mock_s3_deprecated
     def test_get_processed_not_found(self, boto_helper):
-        with mock.patch("socorro.lib.transaction.BACKOFF_TIMES", [0]):
```

Comment:
This made it so that the test would pass faster when raising errors. We don't have `BACKOFF_TIMES` anymore, so I removed these mocks.